src/theme/theme.ts

```diff
@@ -77,19 +80,23 @@ export const defaultPaletteColors = {
   ...allColors,
   brand: allColors.committedYellow,
   primary: allColors.committedGrey,
-  secondary: allColors.committedYellow,
+  secondary: {
+    '300': allColors.committedYellow[50],
+    '500': allColors.committedYellow[200],
+    '700': allColors.committedYellow[400]
+  },
```

Comment:
why is this just the three colours?

Reply:
it should do something like `darken(allColors.committedYellow, 0.5)`

Reply:
I assume this isn't needed?